tensorflow_datasets/core/visualization/image_visualizer.py

```diff
@@ -27,6 +27,7 @@
 from tensorflow_datasets.core import lazy_imports_lib
 from tensorflow_datasets.core.visualization import visualizer
 
+plt = lazy_imports_lib.lazy_imports.matplotlib.pyplot
```

Comment:
`lazy_imports` should not be global.
This will introduce unnecessary dependencies on the user and hence the purpose of `lazy_imports` is defeated.
Please revert back the changes.

Reply:
Okay I will make changes but don't get like why importing them global make trouble, `lazy imports` are used for external dependencies and here in all methods lazy import for `matplotlib `was used ?

Reply:
Consider a case when the `show_example()` of a different visualizer is called.
Assume, this visualizer does not require `matplotlib`.

But because of the global import, the user will get an error if he doesn't have this dependency installed.

Reply:
If I am correct, I make global for Image Visualizer which I think have no connection with other visualizers in future like Video or audio

Reply:
@Eshan-Agarwal, vijayphoenix is right. This line will be executed during `import tensorflow_datasets as tfds` and will crash for all users which don't have this dependency

Reply:
@Conchylicultor thanks for clarify this, changing

Reply:
Try reading the code of `show_examples.py`.